src/App.js

```diff
@@ -1,23 +1,64 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { useState } from 'react';
+import { db } from './lib/firebase.js';
+import { writeToFirestore } from './lib/firebase';
 import './App.css';
 
 function App() {
+  let [results, setResults] = useState([]);
+  let [name, setName] = useState('');
+
+  React.useEffect(() => {
```

Comment:
To be consistent, you could import `useEffect` up above, but no big deal

Reply:
Got it!